python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -91,16 +91,6 @@ def __str__(self):
         return "JobError: %s" % str(self.what)
 
 
-class PreparedStateError(GangaException):
```

Comment:
`PreparedStateError` is not currently used anywhere in the whole code base so I figured it was safe to remove it entirely. If you think we should keep it then I agree, it should be there instead.